notebooks/stress-tests.ipynb

```diff
@@ -38,8 +38,8 @@
      "name": "stdout",
```

Comment:
OK, I moved the URLs out of the stress_test, so that it loops through each URL to run a stress_test on it with 3 threads.
Which results in 4(URL) executions of 100 runs by 3 threads.